pkg/cmd/create/cluster/scenario/create/hub/openstack/install_config.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: v1
+metadata:
+  name: "{{ .managedCluster.name }}"
+baseDomain: {{ .managedCluster.openstack.baseDnsDomain }}
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  replicas: 3
+  platform:
+    openstack:
+      type: {{ .managedCluster.openstack.masterFlavor }}
+compute:
+- hyperthreading: Enabled
+  name: worker
+  replicas: 3
```

Comment:
Can you support parametrization for replicas like in the other install config.
`  replicas: {{ if .managedCluster.worker }}{{ .managedCluster.worker.replicas }}{{ else }}3{{end}}`